docs/src/main/paradox/reference.md

```diff
@@ -154,12 +154,25 @@ Change the interpretation of the `tile_col`'s cell values according to specified
 
 ### rf_resample
 
-    Tile rf_resample(Tile tile, Double factor)
-    Tile rf_resample(Tile tile, Int factor)
-    Tile rf_resample(Tile tile, Tile shape_tile)
+    Tile rf_resample(Tile tile, Double factor, [String method])
+    Tile rf_resample(Tile tile, Int factor, [String method])
+    Tile rf_resample(Tile tile, Tile shape_tile, [String method])
 
+In __SQL__, three parameters are required for `rf_resample`.:
 
-Change the tile dimension. Passing a numeric `factor` will scale the number of columns and rows in the tile: 1.0 is the same number of columns and row; less than one downsamples the tile; and greater than one upsamples the tile. Passing a `shape_tile` as the second argument outputs `tile` having the same number of columns and rows as `shape_tile`. All resampling is by nearest neighbor method.
+    Tile rf_resample(Tile tile, Double factor, String method)
+    Tile rf_resample(Tile tile, Int factor, String method)
+    Tile rf_resample(Tile tile, Tile shape_tile, String method)
+    Tile rf_resample_nearest(Tile tile, Double factor)
+    Tile rf_resample_nearest(Tile tile, Int factor)
+    Tile rf_resample_nearest(Tile tile, Tile shape_tile)
+
+
+Change the tile dimension by upsampling or downsampling. Passing a numeric `factor` will scale the number of columns and rows in the tile: 1.0 is the same number of columns and row; less than one downsamples the tile; and greater than one upsamples the tile. Passing a tile as the second argument resamples such that the output has the same dimension (number of columns and rows) as `shape_tile`. Resampling methods can be one of: nearest_neighbor, bilinear, cubic_convolution, cubic_spline, lanczos, average, mode, median, max, min, or sum.
+
+Note the last six options apply aggregates when downsampling. For example a 0.25 factor and `max` method returns the maximum value in a 4x4 neighborhood.
```

Comment:
Does the GT algorithm handle the edges as you'd expect?

Reply:
i am not 100% sure. not really sure what to say about this exactly. But it is certainly possible that the neighborhood that gets evaluated has a much smaller size. At least based on my reading of things.